staging/src/slime.io/slime/modules/lazyload/controllers/service_cache.go

```diff
@@ -115,8 +96,97 @@ func newSvcCache(clientSet *kubernetes.Clientset) (*NsSvcCache, *LabelSvcCache,
 			}
 			labelSvcCache.Unlock()
 
+			// add eventSvc ports to portProtocolCache again
+			if ns != r.env.Config.Global.IstioNamespace {
+				portProtocolCache.Lock()
+				for _, port := range service.Spec.Ports {
+					p := port.Port
+					if portProtocolCache.Data[p] == nil {
+						portProtocolCache.Data[p] = make(map[Protocol]uint)
+					}
+					switch getProtocol(port) {
+					case ProtocolHTTP:
+						portProtocolCache.Data[p][ProtocolHTTP]++
+					case ProtocolTCP:
+						portProtocolCache.Data[p][ProtocolTCP]++
+					case ProtocolUnknown:
+						portProtocolCache.Data[p][ProtocolUnknown]++
```

Comment:
It looks like this value will keep increasing, right?

Reply:
not support reducing the port here, so only increase. consider to support reducing port in the future, so not use bool value here